suite-native/ethereum-tokens/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { TokenInfo } from '@trezor/blockchain-link';
+
+export const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
```

Comment:
You should not need to export this it's should be always used in selector probably, so you can even place it to selectors file.

Reply:
https://github.com/trezor/trezor-suite/pull/7757/commits/5e37d7d94cb39dbe372e450cf0cc2fbf49ad85d5

Reply:
https://github.com/trezor/trezor-suite/pull/7757/commits/0d472243eaa3c6188b468616b3ee681580f89698